src/polymer/iron-flex-layout-classes.ts

```diff
@@ -162,25 +154,25 @@ ${indent}<style include="${modules}"></style>`)];
     if (!body || !body.__location) {
       return warnings;
     }
-    const missingModules = getMissingStyleModules(parsedDocument.ast);
-    if (!missingModules.length) {
+    const missingModules =
+        getMissingStyleModules(parsedDocument, parsedDocument.ast, warnings);
+    if (!missingModules) {
       return warnings;
     }
-    // TODO(valdrin): update the warning location to be at the spot where
-    // the class is used.
-    const warning = createWarning(parsedDocument, missingModules);
-    const modules = missingModules.map((m) => m.module).join(' ');
+    const warning = warnings[warnings.length - 1];
```

Comment:
Why put this on the last warning? I'd recommend putting it on the first warning instead.